src/routes/(default)/about/+page.svelte

```diff
@@ -9,14 +9,29 @@
 <div class="container">
 	<div class="section">
 		<h1>Vi skaper digitale produkter</h1>
-		<p>Bjerk er et dialogdrevet byrå innen digital produktutvikling og programvareutvikling.</p>
+		<p>
+			Det beste i verden er skapt av de dristige, utfordrere, de som ser ting annerledes som er
+			ambisiøse, nysgjerrige og motiverte nok til å finne orden i virvaret av det uforståelige.
```

Comment:
```suggestion
			Det beste i verden er skapt av de dristige, utfordrere, de som ser ting annerledes som er
			ambisiøse, nysgjerrige og motiverte nok til å finne orden i virvaret av det uforståelige.
```

Not a fan of the double "som" here. It reads poorly even if it's grammatically correct